vendor.conf

```diff
@@ -62,6 +62,7 @@ github.com/docker/libkv                             458977154600b9f23984d9f4b82e
 github.com/vishvananda/netns                        db3c7e526aae966c4ccfa6c8189b693d6ac5d202
 github.com/vishvananda/netlink                      f049be6f391489d3f374498fe0c8df8449258372 # v1.1.0
 github.com/moby/ipvs                                4566ccea0e08d68e9614c3e7a64a23b850c4bb35 # v1.0.1
+github.com/urfave/cli                               a65b733b303f0055f8d324d805f393cd3e7a7904
```

Comment:
:grimacing: fun times

(How many different flag parsers are involved in `moby/moby` post-merge here? :weary:)

Reply:
Yeah, we should move this to cobra. I recall when we moved moby to use Cobra, we created a copy of the `mflags` package to libnetwork.